### ISSUE TITLE
Problems localized coordinates

### ISSUE THREAD

**Issue by rmarabini** [scipion-em/scipion-em-localrec#15]
scipion-em-localrec version 2.0 branch master
scipion version: 2.0 (dioclecian)

Generate coordinates does not seem to be in the proper place.

1) first I created a phantom made with 13 little spheres (12 in the vertex of a icosahedran - sym=i2- and another in the volume center. 
2) after that a generated a set of 216 projection with the command 

scipion xmipp_phantom_project -i  pentoms.feat -o projections.stk --method real_space  --params pentom_10_psi_tilt_theta.xmd

3) these projections has a bit of noise plus shifts in x and y
4) I imported the projections in  relion using the created xmd file
5) I double check that everything was OK reconstructing the phantom (relion_reconstruct)
6) Using this dataset I executed the protocol localized subparticles and then Analyze. For the case in which there was no shifts or the euler angles where zero the results are OK but if this is not the case (see figure) the coordenates do not correspond with the projection of the vertices.

Zero degree case:
![right](https://user-images.githubusercontent.com/10512802/59781161-889c6000-92bb-11e9-9b99-0ac31c729a8b.png)

Arbitrary direction
![wrong](https://user-images.githubusercontent.com/10512802/59781124-76babd00-92bb-11e9-8b7c-c27aaa38bd17.png)

So my conclusion is that the coordinates are non been properly computed.

Attachment:

[pentom.param.txt](https://github.com/scipion-em/scipion-em-localrec/files/3306903/pentom.param.txt)
[pentoms.feat.txt](https://github.com/scipion-em/scipion-em-localrec/files/3306906/pentoms.feat.txt)

**Comment by vabrishami** [comment]
Thanks, Roberto. Could you please send me the parameters used in the protocol? Maybe a snapshot is enough.

**Comment by rmarabini** [comment]
Hi vahid the paramters are
![parameters](https://user-images.githubusercontent.com/10512802/59830870-d27b5980-9340-11e9-859f-90270b588111.png)

Since I generate the phantom I know the position of each vertex and I just select one of them as localization vector

**Comment by vabrishami** [comment]
Hi Roberto,

I think when you generate the projections, the shifts, based on the Xmipp convention, have different signs from what localrec expected. So, I change the shift signs and now it seems ok to me:

![image](https://user-images.githubusercontent.com/9831418/59843922-4548fc80-9362-11e9-9736-1662228df7e1.png)
![image](https://user-images.githubusercontent.com/9831418/59843984-67427f00-9362-11e9-920a-9e60b7e0dd5c.png)

Please close the issue if your convinced :)

**Comment by rmarabini** [comment]
OK let us see.

 1) I generated the projections with xmipp. Therefore, the shifts follow
xmipp conventions
 2) then I imported the data in scipion. A new metadata file was created,
this new metadata  follows scipion conventions
 3) In order to check that the metadata conversion was OK I made a
reconstruction with relion_reconstruct using the imported particles. The
result was correct so I conclude that the metadata information created by
scipion during the importation of the particles is correct.
 4) Finally, I executed localrec. I do not know how locarec protocol is
implemented but it should take scipion metadata, convert it if needed and
finally execute the algorithm. This step failed.

My point is that it is irrelevant how I created the images because after
the importation they follow scipion convention. In order to further
validate the projection generation I am going to import the images, delete
the angles and shift information, execute the protocol relion_refine and
then run localrec. whenever this is done I will let you know.

------

By the way I do not understand why the images you sent support the theory
that changing the sign of the shifts is enough to get the proper results.
The green crosses should be over the white circles and this is not the case.

     cheers

      Roberto
